src/main/resources/react/src/pages/FormRegister/routesRegister.js

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import { Route, Switch } from "react-router-dom";
-import Form from "./Form.js";
+
+import LogIn from './LogIn';
+import SignUp from './SignUp';
 
 export default () => {
   return (
     <Switch>
-      <Route path="/registration/" exact component={Form} />
-      <Route path="/registration/register" component={Form} />
+      <Route path="/logIn/" exact component={LogIn} />
+      <Route path="/logIn/signUp" component={SignUp} />
```

Comment:
You have misspelling into both lines that send request to "logIn". All letters must be low case.

Reply:
Done